From 49432580a83da02f7ccf9b8773fb0ac85a1c8100 Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Thu, 24 Sep 2015 20:03:03 +0800
Subject: [PATCH] MLK-11624 ARM: dts: imx6qp-sabresd: Correct DOVDD supply for
 camera

DOVDD ( GEN_1V8 ) is supplied by SW4 on i.MX6dqp-sabresd board
This patch corrects it.

Signed-off-by: Robby Cai <r63905@freescale.com>
---
 arch/arm/boot/dts/imx6qp-sabresd.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qp-sabresd.dts b/arch/arm/boot/dts/imx6qp-sabresd.dts
index a5d80ac..ffc2e7a 100644
--- a/arch/arm/boot/dts/imx6qp-sabresd.dts
+++ b/arch/arm/boot/dts/imx6qp-sabresd.dts
@@ -29,6 +29,10 @@
 	prefetch;
 };
 
+&ov564x {
+	DOVDD-supply = <&sw4_reg>; /* 1.8v */
+};
+
 &pcie {
 	power-on-gpio = <&gpio3 19 0>;
 	reset-gpio = <&gpio7 12 0>;
-- 
1.8.0

